ecommers/lib/core/provider_models/profile_provider_model.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:ecommers/core/models/edit_profile_model.dart';
+import 'package:ecommers/shared/dependency_service.dart';
+import 'package:flutter/cupertino.dart';
+
+class ProfileProviderModel extends ChangeNotifier {
+  String getEmail() {
```

Comment:
use getter for example String get email =>profileService.user.email;

Reply:
Done